### PR TITLE
refactor: move debug config + debug control out of SceneController

### PR DESCRIPTION
**Pull request by BrianAmadori**
This PR adds some order to our debug flow:

* `DebugController`
Is in charge of the debug business logic, interfaces with `DebugView`

* `DebugView`
Controls the `fpsPanel`, `enginePanel` and `scenePanel` visual details.

* `DebugBridge`
Interfaces between kernel and `DebugController`

* `DataStore.DebugConfig`
Stores the debug internal state, debug is enabled, disabled, etc.

More changes:
- Moved most of these debug classes inside their own assembly
- Renamed some members of `DebugConfig` to reflect the proper function and add docs.
- `FPSPanel`, `EngineDebugPanel` and `SceneDebugPanel` are now in their own prefab, removed them from `SceneController` prefab. On top of that, they are now loaded on runtime like the rest of our HUD views.

### CODE REVIEW

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/GlobalUsersPositionMarker/Handlers/FetchScenesHandler.cs [hunk 1]
```diff
@@ -3,123 +3,127 @@
...
 using System.Collections;
-using UpdateMode = MapGlobalUsersPositionMarkerController.UpdateMode;
+using UpdateMode = DCL.MapGlobalUsersPositionMarkerController.UpdateMode;
 
-/// <summary>
-/// Handle the fetch of hot scenes at intervals.
-/// Interval time may change accordingly if set to update in background or foregraound
-/// </summary>
-internal class FetchScenesHandler : IDisposable
+namespace DCL
```
Comment (BrianAmadori) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/GlobalUsersPositionMarker/Handlers/FetchScenesHandler.cs:8]: No important changes on this file, just added the code inside DCL namespace.

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/GlobalUsersPositionMarker/Handlers/MarkersHandler.cs [hunk 2]
```diff
@@ -3,145 +3,150 @@
 using UnityEngine;
 using Random = UnityEngine.Random;
 
-/// <summary>
-/// Receive a list of hot scenes and handle what markers to show, what markers to hide and places them on the map
-/// </summary>
-internal class MarkersHandler : IDisposable
+namespace DCL
```
Comment (BrianAmadori) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/GlobalUsersPositionMarker/Handlers/MarkersHandler.cs:6]: No important changes on this file, just added the code inside DCL namespace.

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/GlobalUsersPositionMarker/MapGlobalUsersPositionMarkerController.cs [hunk 3]
```diff
@@ -2,88 +2,95 @@
 using System.Collections.Generic;
 using UnityEngine;
 
-public class MapGlobalUsersPositionMarkerController : IDisposable
+namespace DCL
```
Comment (BrianAmadori) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/GlobalUsersPositionMarker/MapGlobalUsersPositionMarkerController.cs:5]: No important changes on this file, just added the code inside DCL namespace.

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/GlobalUsersPositionMarker/Utils/UserPositionMarker.cs [hunk 4]
```diff
@@ -2,57 +2,68 @@
 using System;
 using Variables.RealmsInfo;
 
-/// <summary>
-/// Wrapper class to handle user's marker GameObject
-/// </summary>
-internal class UserPositionMarker : IDisposable
+namespace DCL
```
Comment (BrianAmadori) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/GlobalUsersPositionMarker/Utils/UserPositionMarker.cs:5]: No important changes on this file, just added the code inside DCL namespace.

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/SceneController.cs [hunk 5]
```diff
@@ -854,77 +882,16 @@ public void ReIntegrateIsolatedScene()
 
         //======================================================================
 
-
```
Comment (BrianAmadori) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/SceneController.cs:857]: Code moved to `SceneController`, `SceneBridge` and `SceneView`

**Review by pbosio — APPROVED** [review]

**Review by AjimenezDCL — APPROVED** [review]
Great job! 

Could you elaborate a bit on the use of the namespace `DCL`? When are we using it?

**Review by sandrade-dcl — APPROVED** [review]
It's a very clean and clearer code now, good work!

**Comment by BrianAmadori** [comment]
@AjimenezDCL `DCL` is the namespace we use for our project (as opposed to plugins, etc). It's best to try to put everything into it when possible to avoid name collisions.

**Pull request merged**